lib/Source.php

```diff
@@ -89,7 +89,7 @@ public function detach($params = null, $opts = null)
 
         $id = $this['id'];
         if (!$id) {
-            $class = \get_class($this);
+            $class = static::class;
```

Comment:
I pinned PHP-CS-Fixer's version to avoid this change for the time being. Can you reinstall your dependencies (`rm -rf vendor/ composer.lock && composer update`) and reset the changes to `Source.php` and `StripeObject.php`?

Reply:
Fixed, PTAL